README.md

```diff
@@ -1,67 +1,58 @@
 # Torch XPU Operators*
-===========================
 
-Torch XPU Operators* implements PyTorch ATen operators for Intel GPU devices, aiming to agilely support PyTroch ATen operations and buffer these operations for Intel GPU upstreaming . For more details, refer to [SYCL kernels for ATen Operators RFC](https://github.com/pytorch/pytorch/issues/114835) for more details.
+Torch XPU Operators* implements PyTorch ATen operators for Intel GPU devices, aiming to agilely support PyTorch ATen operations and buffer these operations for Intel GPU upstreaming . For more details, refer to [SYCL kernels for ATen Operators RFC](https://github.com/pytorch/pytorch/issues/114835) for more details.
 
-## 1. Overview
+## Overview
 
  <p align="center">
      <img src="docs/torch_xpu_ops.jpg" width="100%">
  </p>
 
  * SYCL Implementation for XPU Operators: The Operators in this staging branch will finally be upstreamed to PyTorch for Intel GPU.
 
-## 2. Requirements
+## Requirements
 
-#### Hardware Requirements
```

Comment:
Suggest merge Hardware Requirement and Software Requirement, and remove the hardware list from this README. All will redirect to Bundle web link.